packages/metrics-util/package-lock.json

```diff
@@ -9,38 +9,1261 @@
 			"version": "1.137.3",
 			"license": "MIT",
 			"dependencies": {
+				"@instana/core": "1.137.3",
+				"@instana/serverless": "1.137.3",
+				"@instana/shared-metrics": "1.137.3",
 				"node-fetch": "^2.6.1"
 			},
 			"devDependencies": {
 				"no-code2": "2.0.0"
 			}
 		},
+		"node_modules/@instana/core": {
```

Comment:
Looks like a bug in dependabot. They say they can handle lerna repositories and it has worked so far very well.
Maybe it only happens on "upgrade". I'll close the PR. It's not an important update I think